Assets/_BForBoss/_Core/SceneTemplates/WeaponRaceCourseTemplate/WeaponRaceCourse.unity

```diff
@@ -38,7 +38,7 @@ RenderSettings:
   m_ReflectionIntensity: 1
```

Comment:
Wondering why there are changes to the scene templates?

Reply:
So that next time a user makes a weapon race course level they don't have to add this prefab!